multichannel-app/frontend/app/api/webhook-rtc-event/route.ts

```diff
@@ -57,6 +81,42 @@ export const POST = async (req: Request) => {
             case 'app:knocking':
                 onAppKnocking(appKnocking.parse(event));
                 break;
+            case 'audio:say':
+                onAudioSay(audioSayEvent.parse(event), req);
+                break;
+            case 'audio:say:stop':
+                onAudioSayStop(audioSayStopEvent.parse(event), req);
+                break;
+            case 'audio:say:done':
+                onAudioSayDone(audioSayDoneEvent.parse(event), req);
+                break;
+            case 'audio:play':
+                onAudioPlay(audioPlayEvent.parse(event), req);
+                break;
+            case 'audio:play:stop':
+                onAudioPlayStop(audioPlayStopEvent.parse(event), req);
+                break;
+            case 'audio:play:done':
+                onAudioPlayDone(audioPlayDoneEvent.parse(event), req);
+                break;
+            case 'audio:record':
+                onAudioRecord(audioRecordEvent.parse(event), req);
+                break;
+            case 'audio:record:stop':
+                onAudioRecordStop(audioRecordStopEvent.parse(event), req);
+                break;
+            case 'audio:record:done':
+                onAudioRecordDone(audioRecordDoneEvent.parse(event), req);
+                break;
+            case 'audio:asr:done':
+                onAudioAsrDone(audioAsrDoneEvent.parse(event), req);
+                break;
+            case 'audio:asr:record:done':
+                onAudioAsrRecordDone(audioAsrRecordDoneEvent.parse(event), req);
+                break;
+            case 'audio:transcribe:done':
+                onAudioTranscribeDone(audioTranscribeDoneEvent.parse(event), req);
+                break;
```

Comment:
Ideally, we would use a discriminated union here to avoid the double parsing but not sure we have time to get that working, as we had issues before.